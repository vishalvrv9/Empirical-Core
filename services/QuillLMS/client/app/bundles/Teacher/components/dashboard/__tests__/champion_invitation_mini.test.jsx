import React from 'react';
import { shallow } from 'enzyme';

import ChampionInvitationMini from '../champion_invitation_mini';

describe('ChampionInvitationMini component', () => {

  it('should render', () => {
    expect(shallow(<ChampionInvitationMini />)).toMatchSnapshot();
  });

});
