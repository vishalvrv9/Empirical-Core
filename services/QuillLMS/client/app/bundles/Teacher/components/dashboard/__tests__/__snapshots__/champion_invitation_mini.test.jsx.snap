// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ChampionInvitationMini component should render 1`] = `
<div
  className="mini_container results-overview-mini-container col-md-4 col-sm-5 text-center"
>
  <div
    className="mini_content"
  >
    <div
      className="gray-underline"
      style={
        Object {
          "padding": "8px",
        }
      }
    >
      <img
        className="mini-star-icon"
        src="https://assets.quill.org/images/icons/icon-star.svg"
        style={
          Object {
            "height": "27px",
            "position": "absolute",
            "right": "6px",
            "top": "-11px",
            "transform": "rotate(-22deg)",
            "width": "27px",
          }
        }
      />
      <img
        className="mini-star-icon"
        src="https://assets.quill.org/images/icons/icon-star.svg"
        style={
          Object {
            "height": "14px",
            "position": "absolute",
            "right": "25px",
            "top": "-10px",
            "transform": "rotate(-4deg)",
            "width": "14px",
          }
        }
      />
      <h3>
        Share Quill, Earn Free Premium!
      </h3>
    </div>
    <div>
      <img
        src="https://assets.quill.org/images/illustrations/champion.png"
        style={
          Object {
            "height": 48,
            "marginBottom": 11,
            "marginTop": 13,
          }
        }
      />
    </div>
    <p
      style={
        Object {
          "fontFamily": "lucida-grande, adelle-sans, helvetica",
          "lineHeight": "1.57",
          "padding": "0px 15px",
        }
      }
    >
      With the new Quill Referral Program, you can earn rewards for helping fellow educators discover Quill.
    </p>
    <a
      href="/referrals"
      style={
        Object {
          "display": "block",
          "marginBottom": "12px",
        }
      }
    >
      <button
        className="button button-white beta"
        style={
          Object {
            "margin": "15px auto 0px",
            "maxWidth": "233px",
            "width": "100%",
          }
        }
      >
        Earn Rewards
      </button>
    </a>
    <a
      href="/referrals_toc"
      style={
        Object {
          "color": "#027360",
        }
      }
      target="_blank"
    >
      Learn more about the Referral Program &gt;
    </a>
  </div>
</div>
`;
